tools/wltests/android/create_boot_img.sh

```diff
@@ -41,7 +42,13 @@ c_info "Generate BOOT image:"
 c_info "   $BOOT_IMAGE"
 c_info "using this configuration :"
 c_info "  KERNEL                 : $KERNEL"
-c_info "  RAMDISK                : $RAMDISK"
+
+if [ -z "$RAMDISK" ]; then
```

Comment:
Let's keep these consistent and use the (not strictly required) `"${RAMDISK}"` format ?